ngraph/core/include/ngraph/variant.hpp

```diff
@@ -7,6 +7,7 @@
 #include <map>
 #include <string>
 
+#include "openvino/core/any.hpp"
```

Comment:
Is including _any.hpp_ necessary as it is not used after.

Reply:
used for `ov::RTMap`